src/mono/sample/HelloWorld/HelloWorld.csproj

```diff
@@ -2,16 +2,5 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>$(NetCoreAppToolCurrent)</TargetFramework>
-    <EnableTargetingPackDownload>false</EnableTargetingPackDownload>
   </PropertyGroup>
-
-  <!-- Redirect 'dotnet publish' to in-tree runtime pack -->
-  <Target Name="TrickRuntimePackLocation" AfterTargets="ProcessFrameworkReferences">
-    <ItemGroup>
-      <RuntimePack>
-        <PackageDirectory>$(ArtifactsBinDir)microsoft.netcore.app.runtime.$(RuntimeIdentifier)\$(Configuration)</PackageDirectory>
-      </RuntimePack>
-    </ItemGroup>
-    <Message Text="Packaged ID: %(RuntimePack.PackageDirectory)" Importance="high" />
-  </Target>
```

Comment:
@ViktorHofer 

This broke the mono samples.   Am I missing something?  (I did: `./build.sh mono+libs -c Release ` to build)

```
$ make run MONO_CONFIG=Release
../../../.././dotnet.sh publish -c Release -r osx-x64
Microsoft (R) Build Engine version 17.0.0-preview-21302-02+018bed83d for .NET
Copyright (C) Microsoft Corporation. All rights reserved.

  Determining projects to restore...
  All projects are up-to-date for restore.
  Used runtime pack:
  Used runtime pack: /Users/alklig/.nuget/packages/microsoft.aspnetcore.app.runtime.osx-x64/6.0.0-preview.5.21301.17
/usr/local/share/dotnet/sdk/6.0.100-preview.5.21302.13/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.Sdk.FrameworkReferenceResolution.targets(391,5): error NETSDK1112: The runtime pack for Microsoft.NETCore.App.Runtime.Mono.osx-x64 was not downloaded. Try running a NuGet restore with the RuntimeIdentifier 'osx-x64'. [/Users/alklig/work/dotnet-runtime/delta-bugs/src/mono/sample/HelloWorld/HelloWorld.csproj]
make: *** [publish] Error 1
```